packages/neon/neon/lib/src/widgets/unified_search_results.dart

```diff
@@ -129,18 +127,16 @@ class NeonUnifiedSearchResults extends StatelessWidget {
     final Account account,
     final CoreUnifiedSearchResultEntry entry,
   ) {
-    final size = Size.square(IconTheme.of(context).size!);
     if (entry.icon.startsWith('/')) {
       return NeonCachedImage.url(
-        size: size,
+        size: Size.square(IconTheme.of(context).size!),
         url: entry.icon,
         account: account,
       );
     }
 
     if (entry.icon.startsWith('icon-')) {
       return NeonServerIcon(
-        size: size,
```

Comment:
This should still be passed since we don't want the icon to just fill all the available space

Reply:
Bruh I made it default to the current icon size and forgot to also pull those changes in.

Should I pull them in or leave them out for a separate PR?

Reply:
I think putting it in here makes sense, so yes